ipcalc-reverse.c

```diff
@@ -97,7 +98,7 @@ static char hexchar(unsigned int val)
 	abort();
 }
 
-char *calc_reverse_dns6(struct in6_addr *ip, unsigned prefix)
+extern char *calc_reverse_dns6(struct in6_addr *ip, unsigned prefix)
```

Comment:
Why do you add the extern keyword to these functions? It is also unrelated to the commit message.